src/main.rs

```diff
@@ -33,7 +33,7 @@ fn cli() -> Result<(), String> {
 
 fn symbolic_subcommand() -> App<'static> {
     SubCommand::with_name("symbolic")
-        .about("Symbolically execute Yul programs checking for revert reachability")
+        .about("Symbolically execute Yul programs checking for reachability of solc panics")
```

Comment:
`Panic()`?

Reply:
Well solc came up with it, it is still the main use case of that error. I think it's better to be explicit about it

Reply:
What I mean is that there is an EIP about how to interpret revert data and we could just be explicit about it.